reason-react-native/src/components/TextInput.re

```diff
@@ -129,7 +127,7 @@ external make:
     ~onChangeText: string => unit=?,
     ~onContentSizeChange: contentSizeChangeEvent => unit=?,
     ~onEndEditing: editingEvent => unit=?,
-    ~onFocus: focusEvent => unit=?,
+    ~onFocus: Event.targetEvent => unit=?,
```

Comment:
What about `TextInput.onBlur`?

Reply:
I had revised all TextInput Event types but waited for this one to be merged first. Please see #467.

Reply:
Cool! Thanks a lot! 👍